### PR TITLE
Recipe tags

### PR DESCRIPTION
**Pull request by shaundillon**
Added recipe tags list from Recipeasy and styled them out.

picture 153

### CODE REVIEW

**Comment by shaundillon** [comment]
Added do not merge for now while I sort out the horrendous UI

File: public/js/constants/recipeData.js [hunk 1]
```diff
@@ -0,0 +1,737 @@
...
+  'Nut free',
+  'Gluten free',
+  'Dairy free',
+  'Egg free',
+  'Vegetarian',
+  'Vegan'
+];
+
+export const chefs = [
+  'Nigel Slater and Observer readers',
```
Comment (philmcmahon) [inline public/js/constants/recipeData.js]: We should think about whether we want to link these up to contributor tags - e.g. Nigel Slater has a contrbutor tag: https://tagmanager.gutools.co.uk/tag/16196

**Comment by philmcmahon** [comment]
This looks good to me! Might be worth us thinking about how chefs/contributor tags come together at some point (if at all) but not worth bothering about now. 👍

File: public/js/constants/recipeData.js (hunk 1 above)
Comment (shaundillon) [inline public/js/constants/recipeData.js]: I agree, but was just matching the functionality of recipeasy for now. Adding contributors would be a great improvement though

**Pull request merged**